pkg/server/render.go

```diff
@@ -198,3 +270,23 @@ func (ctrl *Controller) loadTree(gi *storage.GetInput, startTime, endTime time.T
 	}
 	return out, nil
 }
+
+// Request Body Interface
+type RenderDiffParams struct {
+	Name  *string `json:"name,omitempty"`
+	Query *string `json:"query,omitempty"`
+
+	From  string `json:"from"`
+	Until string `json:"until"`
+
+	Format   string `json:"format"`
+	MaxNodes *int   `json:"max-nodes,omitempty"`
+
+	Left  RenderTreeParams `json:"left-params"`
+	Right RenderTreeParams `json:"right-params"`
+}
```

Comment:
Taking into account that we already follow [mixed-case naming convention](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format) in responses (e.g., `sampleRate`), perhaps we should do so for requests as well.

Reply:
Sure, I have shifted to camel case.